frontend/src/app/app.module.ts

```diff
@@ -81,11 +83,13 @@ const nonProductionProviders = [{
     NgxJsonViewerModule,
     MatExpansionModule,
     MatProgressBarModule,
+    MatSnackBarModule
   ],
   providers: [
     ...!environment.production ? nonProductionProviders : [],
     HierarchyService,
-    DocumentationService
+    DocumentationService,
+    NotificationService
```

Comment:
Typescript supports trailing commas. It convenient for source control diff because in avoids having the last line of the table modified just to add the comma.